README.adoc

```diff
@@ -104,7 +104,7 @@ Before you begin, build your application. To do this, navigate to the `start` di
 mvn install
 ```
 
-Your [hotspot]`pom.xml` file is already configured to add your REST application to the `defaultServer`. But you can tweak this configuration or add your own for another server by updating the [hotspot=181-192]`<execution/>` element.
+Your [hotspot]`pom.xml` file is already configured to add your REST application to the `defaultServer`. But you can tweak this configuration or add your own for another server by updating the [hotspot=185-197]`<execution/>` element.
```

Comment:
Checked on ogden, the hotspot selects one additional line. So the hotspot should be 185-196, not 185-197